src/scss/about.scss

```diff
@@ -2,15 +2,17 @@
 
 .about {
   width: 100%;
-  height: 100vh;
+  min-height: 100vh;
+  height: 110vh; //высота как на макете + убрали риски не влезть
+  max-height: 115vh;
```

Comment:
Replace this value with pixels. You can't be highly sure what's the height of the browser is